### PR TITLE
Moved shutdown() call.

### PR DESCRIPTION
**Pull request by kapkic**
Moved shutdown() call, should be working correctly.
To test it, add these two lines to serverclient.c after line 73 in unified_syscalls.

```
    shutdown(server_fd,SHUT_RDWR);
    printf("Shutdownsent\n");
```

### CODE REVIEW

File: src/shared/platform/lind_platform.h [hunk 1]
```diff
@@ -69,7 +69,7 @@
 #define LIND_safe_net_getsockname       42
 #define LIND_safe_net_getsockopt        43
 #define LIND_safe_net_setsockopt        44
-#define LIND_safe_net_shutdown          45
+#define LIND_safe_net_shutdown          45 //unchanged, already in location
```
Comment (jesings) [inline src/shared/platform/lind_platform.h]: This comment is unnecessary

File: src/trusted/service_runtime/lind_syscalls.c [hunk 2]
```diff
@@ -745,7 +745,7 @@ StubType stubs[] = {
         {LindCommonPreprocess, 0, 0}, /* 42 LIND_safe_net_getsockname */
         {LindCommonPreprocess, 0, 0}, /* 43 LIND_safe_net_getsockopt */
         {LindCommonPreprocess, 0, 0}, /* 44 LIND_safe_net_setsockopt */
-        {LindCommonPreprocess, 0, 0}, /* 45 LIND_safe_net_shutdown */
+        {0},
```
Comment (jesings) [inline src/trusted/service_runtime/lind_syscalls.c]: Keeping the comment just to `/* 45 */` would probably be preferable

**Review by jesings — COMMENTED** [review]
Some minor comment changes should be made, but this looks great!

File: src/shared/platform/lind_platform.h (hunk 1 above)
Comment (kapkic) [reply src/shared/platform/lind_platform.h]: Removed.

File: src/trusted/service_runtime/lind_syscalls.c (hunk 2 above)
Comment (kapkic) [reply src/trusted/service_runtime/lind_syscalls.c]: Added `/* 45 */` with PR updates commit

**Pull request merged**